.eslintrc.js

```diff
@@ -40,5 +41,29 @@ module.exports = {
         document: "readonly",
       },
     },
+    {
+      files: ["**/cli.js"],
+      rules: {
+        "node/shebang": 0,
+        "no-process-exit": 0,
+      },
+    },
+    {
+      files: ["ssr/**/*.js"],
+      parserOptions: {
+        sourceType: "module",
+        allowImportExportEverywhere: false,
+      },
+      rules: {
+        "node/no-unsupported-features/es-syntax": [
+          "error",
+          {
+            ignores: ["modules"],
+          },
+        ],
+        "node/no-unpublished-import": "off",
+        "no-undef": "off",
```

Comment:
This was for the `arguments` call, and could probably be inline instead

Reply:
What `arguments` call??

Reply:
https://github.com/mdn/yari/blob/a366301d0f61e4c7c7eab6f4385fc43fd7d3b356/ssr/render.js#L53

Reply:
Remove this line and wait for https://github.com/mdn/yari/pull/2747
I tested it and it means we don't need to have this rule.